articles/container-registry/container-registry-oras-artifacts.md

```diff
@@ -52,9 +52,9 @@ If needed, run the [az group create](/cli/azure/group#az-group-create) command t
 ```azurecli
 az group create --name $ACR_NAME --location southcentralus
 ```
-### Create ORAS Artifact enabled registry
+### Create OCI Artifact Manifest enabled registry
 
-Preview support for ORAS Artifacts requires Zone Redundancy, which requires a Premium service tier, in the South Central US region. Run the [az acr create](/cli/azure/acr#az-acr-create) command to create an ORAS Artifacts enabled registry. See the `az acr create` command help for more registry options.
+Preview support for OCI Artifact Manifest requires Zone Redundancy, which requires a Premium service tier, in the South Central US region. Run the [az acr create](/cli/azure/acr#az-acr-create) command to create an OCI Artifact Manifest enabled registry. See the `az acr create` command help for more registry options.
```

Comment:
We no longer require zone redundancy, and we support any more than the south central US region.

> ~~Preview support for OCI Artifact Manifest requires Zone Redundancy, which requires a Premium service tier, in the South Central US region.~~ Run the [az acr create](/cli/azure/acr#az-acr-create) command to create an OCI Artifact Manifest enabled registry. See the `az acr create` command help for more registry options.